gcf_data_mapper/read.py

```diff
@@ -0,0 +1,36 @@
+import csv
+import json
+from typing import Any, Optional
+
+import click
+
+
+def read_csv(file_path: str) -> list[str]:
+    """Reads a csv file and returns a list of values"""
+    with open(file_path, "r") as file:
+        csv_reader = csv.DictReader(file)
+        data = []
+        for line in csv_reader:
+            data.append(line["country"])
+        return data
+
+
+def read_json(file_path: str) -> dict:
+    """Reads a json file, we will just read the file for now and echo a value"""
+    with open(file_path, "r") as file:
+        data = file.read()
+        return json.loads(data)
+
+
+def read_data_file(file_path: str) -> Optional[dict[str, Any] | list[str]]:
+    """Simple program that reads a data file, calls a function to read a csv or json file respectively"""
```

Comment:
Line too long for a docstring - PEP8 guides suggest 72 chars for docstrings. We can enable this as a vertical ruler in your IDE if you don't already have it set up :) Should be a short one liner for something obvious or a multiline one if a more detailed description is needed - see [PEP257](https://peps.python.org/pep-0257/#one-line-docstrings)

Reply:
ahhh it's weird trunk catches this in my frontend applications, but not in this one